lib/src/cli/atlas_apps/baas_client.dart

```diff
@@ -510,6 +530,24 @@ class BaasClient {
     final result = input.replaceRange(4, input.length - 4, '');
     return result;
   }
+
+  Future<void> setAutomaticRecoveryEnabled(String name, bool enable) async {
+    final dynamic docs = await _get('groups/$_groupId/apps');
+    dynamic doc = docs.firstWhere((dynamic d) => d["name"] == "$name$_appSuffix", orElse: () => throw Exception("BAAS app not found"));
+    final appId = doc['_id'] as String;
+    final clientAppId = doc['client_app_id'] as String;
+    final app = BaasApp(appId, clientAppId, name);
+
+    final dynamic services = await _get('groups/$_groupId/apps/$appId/services');
+    dynamic service = services.firstWhere((dynamic s) => s["name"] == "BackingDB", orElse: () => throw Exception("Func 'confirmFunc' not found"));
+    final mongoServiceId = service['_id'] as String;
+    final dynamic configDocs = await _get('groups/$_groupId/apps/$appId/services/$mongoServiceId/config');
+    final dynamic flexibleSync = configDocs['flexible_sync'];
+    flexibleSync["is_recovery_mode_disabled"] = !enable;
+    configDocs['flexible_sync'] = flexibleSync;
+    String data = jsonEncode(<String, dynamic>{'clusterName': configDocs['clusterName'], 'flexible_sync': configDocs['flexible_sync']});
```

Comment:
Why not json encode `configDocs`?

Reply:
It doesn't accept all the properties of the config to be set in the request. It insists only clusterName, flexible_sync, sync, permissions and one more thing to be passed.

Reply:
I see, in that case, there should be no need to update it on line 547. We can just do 

```dart
final data = jsonEncode(<String, dynamic>{'clusterName': configDocs['clusterName'], 'flexible_sync': flexibleSync});
```